src/django_components/util/misc.py

```diff
@@ -130,3 +131,19 @@ def is_glob(filepath: str) -> bool:
 
 def flatten(lst: Iterable[Iterable[T]]) -> List[T]:
     return list(chain.from_iterable(lst))
+
+
+def to_dict(data: Any) -> dict:
```

Comment:
The `Args`, `Kwargs`, etc, classes can be NamedTuple or dataclasses (among other). This utility makes sure that we're able to handle these different classes by normalizing their data to a plain dictionary.